pydatalab/pydatalab/apps/chat/blocks.py

```diff
@@ -81,8 +120,23 @@ def render(self):
                 return
 
         try:
+            model_name = self.data["model"]
+            model_dict = self.data["available_models"][model_name]
+            LOGGER.warning(f"Initializing chatblock with model: {model_name}")
+
+            if "claude" in model_name:
+                self.ChatClient = ChatAnthropic(
+                    anthropic_api_key=os.environ["ANTHROPIC_API_KEY"],
+                    model=model_name,
+                )
+            elif "gpt" in model_name:
```

Comment:
```suggestion
            elif model_name.startswith("gpt"):
```

probably slightly more robust; some OSS models use 'GPT' in the name too